microsoft-edge/extensions-chromium/getting-started/part2-content-scripts.md

```diff
@@ -2,26 +2,28 @@
 title: Create an extension tutorial, part 2
 description: Dynamically inserting a NASA picture below the page body tag by using content scripts.
 author: MSEdgeTeam
-ms.author: msedgedevrel
+ms.author: v-jandrew    
```

Comment:
```suggestion
ms.author: msedgedevrel
```
per global Find in the repo to confirm the convention.  also avoid trailing spaces